src/st/e_collect.h

```diff
@@ -37,23 +37,18 @@ static u8 D_80180D94[] = {0x05, 0x5A, 0x05, 0x5B, 0x05, 0x5C, 0x05, 0x5D, 0x05,
                           0x5E, 0x05, 0x5F, 0x05, 0x60, 0x05, 0x61, 0x00, 0x00};
 static u8 D_80180DA8[] = {0x05, 0x62, 0x05, 0x63, 0x05, 0x64, 0x05, 0x65, 0x05,
                           0x66, 0x05, 0x67, 0x05, 0x68, 0x05, 0x69, 0x00, 0x00};
-static u8 D_80180DBC[] = {0x05, 0x6A, 0x05, 0x6B, 0x05, 0x6C, 0x05, 0x6D};
-static u16 D_80180DC4[] = {0x6E05, 0x6F05, 0x7005, 0x7105, 0};
-static u8 D_80180DD0[] = {
-    0x01, 0x8F, 0x00, 0x00, 0xFC, 0xFC, 0x04, 0xFC, 0xFC, 0x04, 0x04, 0x04,
-    0x80, 0x00, 0x00, 0x00, 0x01, 0x00, 0x02, 0x00, 0x04, 0x00, 0x03, 0x00,
-    0x06, 0x00, 0x05, 0x00, 0x07, 0x00, 0x08, 0x00, 0x09, 0x00, 0x00};
-
-static u16 D_80180DF4[] = {0x0000, 0x000E, 0x000F, 0x0011, 0x0010,
-                           0x0013, 0x0012, 0x0014, 0x0015, 0x0016};
+static u8 D_80180DBC[] = {0x05, 0x6A, 0x05, 0x6B, 0x05, 0x6C, 0x05, 0x6D, 0x05,
+                          0x6E, 0x05, 0x6F, 0x05, 0x70, 0x05, 0x71, 0x00};
+static u8 D_80180DD0[] = {0x01, 0x8F, 0x00, 0x00, 0xFC, 0xFC, 0x04,
+                          0xFC, 0xFC, 0x04, 0x04, 0x04, 0x80, 0x00};
+static u16 aluric_subweapons_idx[] = {1, 2, 4, 3, 6, 5, 7, 8, 9};
```

Comment:
Naming convention? https://github.com/Xeeynamo/sotn-decomp/blob/master/docs/STYLE.md#naming

Reply:
I think we have been using `g_` on global and `snake_case` for static? At least for the `src/pc` part.

I'd largely ignore that styling doc as it was created by someone who is not contributing any more and it's largely ignored anyway

Reply:
If we aren't following it should we just remove it? I've been directing contributors to it since I thought it was still being enforced

Reply:
If we're considering removing it, IMO we should instead consider updating it so that it can still be a single point of reference for anyone joining the project.

Reply:
I think we should keep the document, not enforce it on new PRs, and perhaps add a line to it saying "If you find sections of code in the repo which do not fit this style, feel free to submit a PR to bring them into alignment with this". That way it doesn't become a hangup, and style-cleanups can become another way for people to contribute if they want to.

Reply:
Okay thanks for the feedback. I'll go ahead and merge this and follow-up tweaking the STYLE.md document